Tensile/KernelWriter.py

```diff
@@ -199,11 +227,10 @@ def makeSchedule(self, kernel, tensorParametersA, tensorParametersB, localWriteE
         writesToSched = self.localWriteACode.countType(Code.LocalWriteInst) + \
                      self.localWriteBCode.countType(Code.LocalWriteInst)
       startIter = kernel["LoopIters"] - writesToSched
-      # to offset last LW 1 mfma with barrier
-      # TODO: how many mfma to offset based on mfma latency
-      if kernel["EnableMatrixInstruction"]:
-        writesToSched += 1
```

Comment:
This change affects cases for SIA < 3, should there be another case for this?

Reply:
the purpose of this is to control number of mfma between last LW and barrier,  but it will wrongly become the number of iteration between last LW and barrier for SIA1.
for SIA2, since it will schedule LW b2b, this won't affect SIA2.